src/azure-cli-core/azure/cli/core/commands/progress.py

```diff
@@ -195,3 +195,13 @@ def update_progress(self):
 
     def end(self):
         self.hook.end()
+
+
+class PollerProgressBar(IndeterminateProgressBar):
```

Comment:
@jiasli could you please help take a look?

Reply:
Whether this `PollerProgressBar` needs to be included in the core module depends on whether it is a potential common requirement or just a special usage of AKS

Reply:
I'm currently working on a fix for this to not this `PollerProgressBar` and instead use the lower level `ProgressView` and `ProgressHook` methods instead. These new classes don't actually add anything new (they serve as high-level wrappers) and simply pollute the core libraries in my opinion.

Reply:
Addressed, we no longer create an entirely new class for this in the `azure.cli.core` libraries.